lib/widgets/consult_pad.dart

```diff
@@ -232,12 +288,31 @@ class _ConsultPadWidgetState extends State<ConsultPadWidget> {
       ),
     );
   }
+  SlidableAction _removeInvestigationAction(OmrsOrder investigation){
+    return SlidableAction(
+        onPressed: (context){
+          setState(() {
```

Comment:
Is this required? if you notice _showConsultation() anyway is built within a consumer<ConsultationBoard> .. so anytime you are updating ConsultationBoard, the build for that section will get triggered.

Reply:
Fixed it.